www/lib/mailer.php

```diff
@@ -177,34 +177,30 @@ function webhook_trigger($eventName, $deviceId = null, $payload = array()) {
 $dow = array('M', 'T', 'W', 'R', 'F', 'S', 'U');
 
 #parse start timestamp into dow/hour/minute
-$p['day'] = $dow[date('w', $timestamp)-1];
+$p['day'] = $dow[(date('w', $timestamp) + 6) % 7];
 $p['hour'] = date('G', $timestamp);
 $p['min'] = date('i', $timestamp);
 
-
 #form a query and select applicable rules
 $query = "SELECT * FROM notificationSchedules 
-	WHERE ";
-if (isset($device_id))
-$query .= "
-		cameras LIKE '%|$device_id|%' 
-	AND ";
-$query .= "
-		day LIKE '%{$p['day']}%' 
-	AND 
-	(
-		(s_hr = {$p['hour']} AND s_min < {$p['min']}) 
-		OR 
-		(e_hr = {$p['hour']} AND e_min > {$p['min']}) 
-		OR 
-		(s_hr < {$p['hour']} AND e_hr > {$p['hour']})
-	)
-	AND
-		disabled = 0
+    WHERE ";
+if (isset($device_id)) {
+    $query .= " cameras LIKE '%|$device_id|%' AND ";
+}
+$query .= " day LIKE '%{$p['day']}%' AND 
+    (
+        (s_hr = {$p['hour']} AND s_min < {$p['min']}) 
+        OR 
+        (e_hr = {$p['hour']} AND e_min > {$p['min']}) 
+        OR 
+        (s_hr < {$p['hour']} AND e_hr > {$p['hour']})
+    )
+    AND disabled = 0
 ";
 $rules = data::query($query);
-if (!$rules){
-	exit('I: No matching rules');
+
+if ($!rules) {
```

Comment:
This is a syntax error.

```suggestion
if (!$rules) {
```